### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by GeneTel**
Completed 2.1 ,don't  know how to do 2.2 and 2.3, will have to move to the next lesson that i have not started yet

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 public class Main {
-    static void showMessage(String message){
-        System.out.println(message);
-    }
+
+
     public static void main(String[] args) {
-       showMessage("Hi");
-Random random = new Random();
+
```
Comment (DarynaHorobei97) [inline src/Main.java:5]: Create a new method out of the main method but in the middle of your class. The main method is needed for running your methods, for their usage.

Correct method implementation:

static int method(int i) {
        int sumOfNumbers = i % 10 + (i / 10) % 10 + i / 100;
        System.out.println(sumOfNumbers);
        return sumOfNumbers ;
    }

Then you can use the method in the main method:
method(225);

Please find out with it attentively

Comment (DarynaHorobei97) [reply src/Main.java:5]: 2.2 implementation:
   static void method(String word1, String word2) {
        String concatStrings = word1+ word2;
        int lengthOfTwoWords = concatStrings.length();
        System.out.println("The sum of characters for provided 2 words: " + lengthOfTwoWords);
    }

Then you can use the method in the main method:
method("Hello, ", "Gene");

Comment (DarynaHorobei97) [reply src/Main.java:5]: 2.3 implementation:
    static boolean method(int number) {
        if (number > 0) {
            System.out.println("Your number is larger than 0");
            return true;
        } else {
            System.out.println("Your number is not larger than 0");
            return false;
        }
    }

Then you can use the method in the main method:
method(5);

**Review by DarynaHorobei97 — COMMENTED** [review]
Please find out methods implementation attentively. I left answers for you

File: src/Main.java (hunk 1 above)
Comment (GeneTel) [reply src/Main.java:11]: Fixed, did not figure out how to overload 3.3

**Pull request merged**